_posts/fr/newsletters/2023-05-03-newsletter.md

```diff
@@ -0,0 +1,201 @@
+  et de libérer de la mémoire pour chaque UTXO individuellement. Les
+  consultations d'UTXO représentent une part importante des accès à la
+  mémoire, en particulier pendant l'IBD. Les analyses comparatives indiquent
+  que la réindexation est accélérée de plus de 20 % grâce à la gestion plus
+  efficace de la mémoire.
+
+- [Bitcoin Core #25939][] permet aux nœuds avec l'index de transaction
+  optionnel activé de rechercher cet index lors de l'utilisation de la
+  RPC `utxoupdatepsbt` pour mettre à jour un [PSBT][topic psbt] avec des
+  informations sur les sorties de transaction qu'il dépense. Lorsque la
+  RPC a été mise en œuvre pour la première fois en 2019 (voir [Bulletin #34][news34 psbt]),
+  deux types de sorties étaient courants sur le réseau : les sorties legacy
+  et les sorties segwit v0. Chaque dépense d'une sortie ancienne dans une
+  PSBT doit inclure une copie complète de la transaction qui contenait
+  cette sortie afin qu'un signataire puisse vérifier le montant de cette
+  sortie. Créer une dépense sans vérifier le montant de la sortie dépensée
+  peut conduire le dépenseur à surpayer massivement les frais de transaction,
+  d'où l'importance de la vérification.
+
+  Chaque dépense d'une sortie segwit v0 s'engage sur son montant afin
+  de permettre aux PSBT d'inclure uniquement la clé scriptPubKey et le
+  montant de la sortie plutôt que l'ensemble de la transaction précédente.
+  On pensait ainsi éliminer la nécessité d'inclure la totalité de la
+  transaction. Puisque chaque sortie de transaction non dépensée pour
+  chaque transaction confirmée est stockée dans l'ensemble UTXO de Bitcoin
+  Core, la RPC `utxoupdatepsbt` peut ajouter la clé de script et le montant
+  nécessaires à n'importe quelle PSBT dépensant une UTXO. La RPC `utxoupdatepsbt`
+  recherchait également auparavant des UTXO dans le mempool du nœud local
+  pour permettre aux utilisateurs de dépenser les résultats de transactions
+  non confirmées.
+
+  Cependant, après que `utxoupdatepsbt` a été ajouté à Bitcoin Core, plusieurs
+  dispositifs de signature matérielle ont commencé à exiger que même les spends
+  des sorties segwit v0 incluent des transactions complètes afin d'éviter une
+  variante de surpaiement de frais qui pourrait résulter d'un utilisateur signant
+  apparemment deux fois la même transaction (voir [Bulletin #101][news101 overpayment]).
+  Cela a renforcé la nécessité de pouvoir inclure des transactions complètes
+  dans un PSBT.
+
+  Ce PR fusionné recherchera dans l'index des transactions (s'il est activé)
+  et dans le mempool du nœud local les transactions complètes, et les inclura
+  dans le PSBT si nécessaire. Si une transaction complète n'est pas trouvée,
+  l'ensemble UTXO sera utilisé pour les dépenses des sorties segwit. Notez
+  que Taproot (segwit v1) élimine le problème de surpaiement pour la plupart
+  des transactions qui dépensent au moins une sortie taproot, nous nous
```

Comment:
The build is triggering on duplicate word here "nous nous". You can keep it (if it is correct to have it twice) but youll need to add "<!-- skip-duplicate-words-test -->" at the beginning of this line.

Reply:
Ok, done for tips ""
but I still have the problem of missing files

Reply:
@Copinmalin sorry github stripped the previous comment. The duplicate line will need to be:

`<!-- skip-duplicate-words-test -->`

Reply:
Done !